Makefile

```diff
@@ -88,13 +88,20 @@ help:
 all: runtime oqc oqd mlir frontend
 catalyst: runtime dialects frontend
 
+# Flag for verbose pip install output
+PIP_VERBOSE_FLAG :=
+
+ifeq ($(VERBOSE),1)
+    PIP_VERBOSE_FLAG := --verbose
+endif
+
```

Comment:
No problem for me at the current location, but wondering if we want it here, or at the start where the other env-vars are defined?

Reply:
Good point, I've moved it to the top with the other env vars in 5eda314d.